sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdServiceEndpoint.java

```diff
@@ -601,7 +604,13 @@ private RntbdRequestRecord writeWhenConnected(
                 HttpConstants.SubStatusCodes.TRANSPORT_GENERATED_410
             );
 
-            BridgeInternal.setRequestHeaders(goneException, requestArgs.serviceRequest().getHeaders());
```

Comment:
why are we not setting the request headers anymore?

Reply:
This is a miss. Will follow up in the next iteration.